src/components/WalletModal/index.tsx

```diff
@@ -1,528 +1,236 @@
-      );
-    } else {
-      return (
-        <HeaderRow>
-          <HoverText>{t('walletModal.connectToWallet')}</HoverText>
-        </HeaderRow>
-      );
     }
-  };
-
-  const renderContent = () => {
-    const isXDEFI = window.xfi && window.xfi.ethereum && window.xfi.ethereum.isXDEFI;
-    const supportsAddNetwork = isMetamask || isCbWallet || isXDEFI || isTalisman || isBitKeep;
-
-    if (web3Error) {
-      return (
-        <ContentWrapper>
-          {web3Error instanceof UnsupportedChainIdError ? (
-            <>
-              <h5> {t('walletModal.pleaseConnectAvalanche')}</h5>
-              {supportsAddNetwork && (
-                <Button variant="primary" onClick={addAvalancheNetwork}>
-                  {t('walletModal.switchAvalanche')}
-                </Button>
-              )}
-            </>
-          ) : (
-            `${t('walletModal.errorConnectingRefresh')}`
-          )}
-        </ContentWrapper>
-      );
-    } else {
-      return (
-        <ContentWrapper>
-          {walletView === WALLET_VIEWS.PENDING ? (
-            <PendingView
-              option={selectedOption}
-              connector={pendingWallet}
-              error={pendingError}
-              setPendingError={setPendingError}
-              tryActivation={tryActivation}
-            />
-          ) : (
-            <>
-              <Box mt="5px" width="100%" mb="5px">
-                <ToggleButtons
-                  options={[CHAIN_TYPE.EVM_CHAINS, CHAIN_TYPE.NON_EVM_CHAINS]}
-                  value={walletType}
-                  onChange={(value) => {
-                    setWalletType(value);
-                  }}
-                />
-              </Box>
-              <Box height="300px">
-                <Scrollbars>
-                  <OptionGrid>{getOptions()}</OptionGrid>
-                </Scrollbars>
-              </Box>
-            </>
-          )}
-        </ContentWrapper>
-      );
-    }
-  };
-
-  function getModalContent() {
-    return (
-      <UpperSection>
-        <ModalWrapper>
-          {renderHeader()}
-
-          <CloseButton onClick={closeModal} />
-        </ModalWrapper>
-
-        {renderContent()}
-      </UpperSection>
-    );
   }
 
   return (
-    <Modal
-      isOpen={walletModalOpen}
-      onDismiss={() => {
-        closeModal();
-      }}
-    >
-      <Wrapper background={background}>{getModalContent()}</Wrapper>
+    <Modal isOpen={open} onDismiss={closeModal}>
+      <Wrapper>
+        <Header>
+          <Text color="text1" fontSize="24px" fontWeight={700}>
+            {t('walletModal.connectToWallet')}
+          </Text>
+          <CloseButton onClick={closeModal} size="24px" />
+        </Header>
+        <Inputs>
+          <TextInput
+            addonAfter={<Search color={theme.text1} size={28} />}
+            onChange={handleSearch}
+            value={searchQuery}
+            placeholder="Try searching for chains"
+          />
+          <ToggleButtons
```

Comment:
this happens because we do not have the same chains for testnet and mainnet because even if there is hedera mainnet and testnet they are separate objects